server/models/bill.coffee

```diff
@@ -13,6 +13,10 @@ module.exports = Bill = cozydb.getModel 'Bill',
     fileId: String
     content: String
     isRefund: Boolean
+    clientId: String # Client number in vendor CRM
+    number: String # Bill Id, in vendor CRM
```

Comment:
The number is really one of the basic [mandatory fields of a Bill](https://www.service-public.fr/professionnels-entreprises/vosdroits/F31808#fiche-item-1) . 
Number is shorter than vendorBillId ; but if you really think it's better, I can switch to it.

To make it shorter, we could alos use idVendor  (ie bill.idVendor )? Short and explicit ?